mmdet/models/roi_heads/scnet_roi_head.py

```diff
@@ -0,0 +1,471 @@
+import torch
+import torch.nn.functional as F
+
+from mmdet.core import bbox2result, bbox2roi
+from ..builder import HEADS, build_head, build_roi_extractor
+from .cascade_roi_head import CascadeRoIHead
+
+
+@HEADS.register_module()
+class SCNetRoIHead(CascadeRoIHead):
+    """RoIHead for SCNet https://arxiv.org/abs/2012.10150.
+
+    Args:
+        num_stages (int): number of cascade stages.
+        stage_loss_weights (list): loss weight of cascade stages.
+        semantic_roi_extractor (dict): config to init semantic roi extractor.
+        semantic_head (dict): config to init semantic head.
+        feat_relay_head (dict): config to init feature_relay_head.
+        glbctx_head (dict): config to init global context head.
+    """
```

Comment:
I think there is one missing [comment](https://github.com/open-mmlab/mmdetection/pull/4356#discussion_r554665621).

Reply:
I think i have resolved the that comment. I only inherit the config HTC including he backbone, rpn, neck, train/test configs, data pipeline except for RoIHead. The SCNetRoIHead use very differrent design and ``forward()`` logics from HTCRoIHead. Besides, SCNetRoIHead does not have ``interleave`` and ``mask_info_flow`` as in HTC.

Reply:
I see. Thanks for the explain.